packages/muelu/test/unit_tests/Smoothers/BlockedDirectSolver.cpp

```diff
@@ -439,7 +439,7 @@ namespace MueLuTests {
       Teuchos::ArrayRCP<const Scalar> xdata = XX->getData(0);
       bool bCheck = true;
       for(size_t i=0; i<XX->getLocalLength(); i++) {
-        if (i>=0  && i< 5) { if(xdata[i] != (SC) 1.0) bCheck = false; }
+        if (i< 5) { if(xdata[i] != (SC) 1.0) bCheck = false; }
```

Comment:
Preferred "promotion to SC" form is `SC (1.0)`.

Reply:
Yeah, I would probably go with `Teuchos::as<SC>(1.0)` or actually `Teuchos::ScalarTrait<SC>::one()`, but this does not bug me enough to fire the auto-tester a seond time x)